lib/HalModuleParser.js

```diff
@@ -90,7 +90,8 @@ HalModuleParser.prototype = {
 	 * @returns Promise
 	 */
 	parseBuffer: function(fileInfo, callback) {
-		if (!Buffer.isBuffer(fileInfo.fileBuffer)) {
+		// todo - ensure that this is not destructive
+		if (!Buffer.isBuffer(fileInfo.fileBuffer) || fileInfo.fileBuffer.length < 128) {
```

Comment:
Why 128? If there is a need for a minimal size check, a better way is probably to use `HEADER_SIZE` defined in `ModuleInfo` and perhaps add `FOOTER_SIZE` or something like that and use the combination of those instead of a magic number.

Reply:
It was just a quick back of the envelope guess to prevent the tests from throwing exceptions. I'll add some more symbols then.